rest/src/main/java/discord4j/rest/service/ChannelService.java

```diff
@@ -129,6 +129,12 @@ public ChannelService(Router router) {
                 .exchange(getRouter());
     }
 
+    @Experimental
+    public Mono<Void> crosspostMessage(long channelId, long messageId) {
```

Comment:
Unsure if I like 'crosspostMessage' or 'publishMessage'

Reply:
same here...
The endpoint and flag are crosspost but in client its publish.
Maybe can be crosspost for endpoints and rename in Message.java to publish (?